credentials_sample.txt

```diff
@@ -0,0 +1 @@
+cluster1.qumulo.com admin admin 8000
```

Comment:
Also just an example, not a real one. 


On Jun 16, 2016, at 2:27 PM, Michael Murray <notifications@github.com> wrote:


In 
credentials_sample.txt:
> @@ -0,0 +1 @@
> +cluster1.qumulo.com admin admin 8000

@carlilek likewise -- specific to your install. Please remove from PR, thanks.
—
You are receiving this because you were mentioned.
Reply to this email directly, 
view it on GitHub, or 
mute the thread.